clusterloader2/testing/experiments/ignore_known_gce_container_restarts.yaml

```diff
@@ -0,0 +1 @@
+IGNORE_RESTARTS_OF: prometheus-to-sd-exporter,metadata-proxy,kube-proxy,fluentd-gcp
```

Comment:
I would like to avoid situation where we add something here and it stays as ignored forever.

Could you add (in a comment) a sentence about each container (why it's being ignored) and/or link to github issue?